README.md

```diff
@@ -39,6 +39,22 @@ assert o.unwrap_or_else(str.upper) == 'yay'
 assert n.unwrap_or_else(lambda: 'default') == 'default'
 ```
 
+There are some methods that support conversion from a `Maybe` to a `Result` type
+in the [result library](https://github.com/rustedpy/result/).  If you wish to
+leverage these methods, you must add `result` to your dependency list:
```

Comment:
I suggest we avoid any of this because there are too many tools to try to cover, so my intention with "you must add 'result' to your dependency list," is literally for the user to simply add it however they would add any other dependency to their project (i.e., just like they added 'maybe'), not for the user to use 'maybe[result]' to make use of the "extras".

However, I agree that it's worth adding something more clear and obvious, so how about just adding a bit more to what you have above for how to install 'maybe'? Specifically, just adding something about, "if you want to also make use of methods for converting Maybes to Results, also pip install result" (I'll write it out a bit more clearly than that).

Reply:
Sorry, that is to say, I agree with your additional comment about the installation section. I'll add info there.

Reply:
Since we have a `pip install` there, maybe we can keep it simple and just provide an example of installing the extra result package with pip only.